### PR TITLE
HPCC-26118 Remove plugin references to environment.conf

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: plugins/py3embed/py3embed.cpp [hunk 1]
```diff
@@ -426,7 +426,15 @@ static class Python3xGlobalState
         PyEval_InitThreads();
         preservedScopes.setown(PyDict_New());
         tstate = PyEval_SaveThread();
-        skipPythonCleanup = queryEnvironmentConf().getPropBool("skipPythonCleanup", true);
+        skipPythonCleanup = true;
+        const char* skipCleanupText = getenv("SKIP_PYTHON_CLEANUP");
+        if (skipCleanupText)
+            skipPythonCleanup = strToBool(skipCleanupText);
+        else
+            skipPythonCleanup = true;
```
Comment (ghalliday) [inline plugins/py3embed/py3embed.cpp:434]: trivial - not needed - already the default value

File: plugins/javaembed/javaembed.cpp [hunk 2]
```diff
@@ -930,6 +932,23 @@ static class JavaGlobalState
         newPath.append(".");
         optionStrings.append(newPath);
 
+        // Options we should set (but allow for override with jvmoptions below)
+        optionStrings.append("-XX:-UseLargePages");
+
+#ifdef _CONTAINERIZED
+        const char *jvmLibPath = getenv("JAVA_LIBRARY_PATH");
```
Comment (ghalliday) [inline plugins/javaembed/javaembed.cpp:939]: is this #if code also useful in non-container?

**Review by ghalliday — COMMENTED** [review]
@richardkchapman a couple of comments, but would be happy to merge as is.  Are these options documented anywhere?

**Comment by richardkchapman** [comment]
There is a Jira to update the docs

File: plugins/javaembed/javaembed.cpp (hunk 2 above)
Comment (richardkchapman) [reply plugins/javaembed/javaembed.cpp:939]: Not sure it's worth adding. Environment vars are harder to set in bare-metal than environment.conf values

**Review by ghalliday — APPROVED** [review]

**Pull request merged**